### PR TITLE
Add new config: sealAllMethods

### PR DESCRIPTION
**Pull request by olleharstedt**

### CODE REVIEW

File: tests/MagicMethodAnnotationTest.php [hunk 1]
```diff
@@ -789,4 +789,31 @@ class B extends A {}
...
     }
+
+    /**
+     * @return void
+     */
+    public function testSealAllMethods()
+    {
+        Config::getInstance()->seal_all_methods = true;
+
+        $this->addFile(
```
Comment (olleharstedt) [inline tests/MagicMethodAnnotationTest.php:800]: Can I add more files in the same method? There are a number of other cases I'd like to test.

Comment (muglug) [reply tests/MagicMethodAnnotationTest.php:800]: Just create separate tests!

**Comment by olleharstedt** [comment]
@muglug I need help to sort out failing Travis:

> ERROR: UndefinedMethod - tests/MagicMethodAnnotationTest.php:816:16 - Method Psalm\Tests\MagicMethodAnnotationTest::expectexceptionmessagematches does not exist (see https://psalm.dev/022)
> $this->expectExceptionMessageMatches('/\b' . \preg_quote($error_message, '/') . '\b/');

Otherwise I've added the tests I want right now. The PR is ready for review.

**Comment by muglug** [comment]
@olleharstedt use `$this->expectExceptionMessage(` instead

**Comment by muglug** [comment]
Thanks!

**Pull request merged**